mlir/include/mlir/Dialect/Vector/IR/VectorOps.td

```diff
@@ -1655,6 +1655,7 @@ def Vector_LoadOp : Vector_Op<"load"> {
     strides. Only unit strides are allowed along the most minor memref
     dimension. These constraints guarantee that elements read along the first
     dimension of the slice are contiguous in memory.
+    Non-unit strides are allowed when doing 0-rank or 1-element vector load.
```

Comment:
[nit] 
```suggestion
    strides. When loading more than 1 element, only unit strides are allowed along the most minor memref
    dimension. These constraints guarantee that elements read along the first
    dimension of the slice are contiguous in memory.
```

Rather than adding a sentence at the end (and splitting one condition across multiple sentences), why not just refine the current wording? Hope I'm not splitting hairs here 😅

Reply:
yeah, that's better, probably.